tests/types/resolving/test_union_pipe.py

```diff
@@ -0,0 +1,176 @@
+import sys
```

Comment:
Rename the file to `test_unions_pipe.py` (or maybe `test_union_pipes.py`) for plurality consistency with the other files